runtime/ops/os.rs

```diff
@@ -55,6 +55,12 @@ fn op_set_env(
 ) -> Result<Value, AnyError> {
   let args: SetEnv = serde_json::from_value(args)?;
   state.borrow::<Permissions>().check_env()?;
+  let invalid_key =
+    args.key.is_empty() || args.key.contains(&['=', '\0'] as &[char]);
```

Comment:
Its probably more accurate to do this:
```suggestion
    args.key.is_empty() || args.key.as_bytes().contains(&[b'=', b'\0']);
```
to be explicit that we are checking against literal bytes (chars in rust are full Unicode Scalar Values, and are 4 bytes)

Reply:
Oh this doesnt support the slice of bytes so it would need to have multiple calls to contains, so might not be worth it

Reply:
I think there is no big difference.